src/main/webapp/help-targetGroupId.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    (Optional) The ID of the group into which the VDB will be provisioned. If unset, a group is selected randomly on the
```

Comment:
> The ID of the group into which...

We have to be careful because "group" can now refer to a VDB Group which I don't think this is referring to. 

I think just adding "engine's dataset group" would clarify. So...

> (Optional) The ID of the engine's dataset group into which the VDB will be provisioned.

Reply:
Separately, "a group is selected randomly". That doesn't seem right. I think it's based on the source ID, no?